installer/templates/phx_live/templates/layout/root.html.leex

```diff
@@ -13,8 +13,8 @@
       <section class="container">
         <nav role="navigation">
           <ul>
-            <li><a href="https://hexdocs.pm/phoenix/overview.html">Get Started</a></li><%= if live and dashboard do %>
-            <li><%%= link "LiveDashboard", to: Routes.live_dashboard_path(@conn, :home) %></li><% end %>
+            <li><a href="https://hexdocs.pm/phoenix/overview.html">Get Started</a></li><%= if dashboard do %><%%= if function_exported?(Routes, :live_dashboard_path, 2) do %>
+            <li><%%= link "LiveDashboard", to: Routes.live_dashboard_path(@conn, :home) %></li><%% end %><% end %>
```

Comment:
Would this indent better?

```suggestion
            <li><a href="https://hexdocs.pm/phoenix/overview.html">Get Started</a></li>
            <%= if dashboard do %><%%= if function_exported?(Routes, :live_dashboard_path, 2) do %>
              <li><%%= link "LiveDashboard", to: Routes.live_dashboard_path(@conn, :home) %></li>
            <%% end %><% end %>
```

Reply:
Selfishly I want to drop the leading spaces so the rendered HTML is _also_ indented properly, but yes this looks better :) :)